src/index.ts

```diff
@@ -872,7 +874,9 @@ function FlatpickrInstance(
       get: () => self.__hideNextMonthArrow,
       set(bool: boolean) {
         if (self.__hideNextMonthArrow !== bool)
-          self.nextMonthNav.style.display = bool ? "none" : "block";
+          if (self.config.showDisabledArrows) {
+            toggleClass(self.nextMonthNav, "disabled", bool);
```

Comment:
This should be extracted out of the if condition and the else block should be removed.